### ISSUE TITLE
TypeError: unsupported operand type(s) for -: 'tuple' and 'float'

### ISSUE THREAD

**Issue by uriva** [googleapis/google-auth-library-python#403]
```
1119 /usr/local/lib/python3.7/site-packages/nlu/bucket_utils.py:23: in load_item
1120     pickled_cache_string = blob.download_as_string()
1121 /usr/local/lib/python3.7/site-packages/google/cloud/storage/blob.py:802: in download_as_string
1122     raw_download=raw_download,
1123 /usr/local/lib/python3.7/site-packages/google/cloud/storage/blob.py:719: in download_to_file
1124     transport, file_obj, download_url, headers, start, end, raw_download
1125 /usr/local/lib/python3.7/site-packages/google/cloud/storage/blob.py:643: in _do_download
1126     download.consume(transport)
1127 /usr/local/lib/python3.7/site-packages/google/resumable_media/requests/download.py:151: in consume
1128     result = _helpers.http_request(transport, method, url, **request_kwargs)
1129 /usr/local/lib/python3.7/site-packages/google/resumable_media/requests/_helpers.py:136: in http_request
1130     return _helpers.wait_and_retry(func, RequestsMixin._get_status_code, retry_strategy)
1131 /usr/local/lib/python3.7/site-packages/google/resumable_media/_helpers.py:150: in wait_and_retry
1132     response = func()
1133 /usr/local/lib/python3.7/site-packages/google/auth/transport/requests.py:252: in request
1134     self.credentials.before_request(auth_request, method, url, request_headers)
1135 /usr/local/lib/python3.7/site-packages/google/auth/transport/requests.py:89: in __exit__
1136     self.remaining_timeout = self._timeout - elapsed
1137 E   TypeError: unsupported operand type(s) for -: 'tuple' and 'float'
```

Did I do something stupid or there's a bug in current version?

**Comment by busunkim96** [comment]
@uriva There's a bug! Thank you for opening an issue.

It'll be fixed in 1.8.1 (release in progress: #402). You can pin to 1.7.2 for now.

**Comment by busunkim96** [comment]
1.8.1 has been released. Will open a follow-up issue for investigating what happened here.
